src/Calendar/Calendar.js

```diff
@@ -139,6 +139,7 @@ export class Calendar extends Component {
         if (!this.props.enableRangeSelection) {
             this.setState({
                 currentDateDisplayed: date,
+                currentYear: date,
                 selectedDate: date,
```

Comment:
Is it strange that we have 3 different state variables (`currentDateDisplayed`, `currentYear`, `selectedDate`) that are all being set to the same value?  Seems like they should either be set to different values or the code that depends on them should just reference a single stored value.

Reply:
@greg-a-smith I agree, but I think it might be better to save that work for an overall refactoring effort instead of putting more time into this. Thoughts?

Reply:
@jbadan Agreed.  That's probably a much larger effort.

Reply:
I agree we should refactor this control. as a side not i did remove the currentYear property and updated the code and tests to work off the currentDateDisplayed property instead 😄